### PR TITLE
🐛 Fixed filter bug & ➕ Re-added the scrolling fix

### PR DESCRIPTION
**Pull request by Etesam913**
## Updated Video

 

https://user-images.githubusercontent.com/55665282/222562695-5d3c746f-5960-44d0-a250-860c2608ef69.mp4

### CODE REVIEW

File: client/src/components/ExploreSidebar.js [hunk 1]
```diff
@@ -1,60 +1,65 @@
...
-    height: 100%;
-    display: grid;
-    grid-column-gap: 5%;
-    grid-template-columns: 47.5% 47.5%;
-`
+  margin: 0 auto;
+  padding: 0 5%;
+  margin-top: 2vh;
+  overflow: scroll;
+  max-height: 70vh;
```
Comment (Etesam913) [inline client/src/components/ExploreSidebar.js:13]: This is for the new scroll container

File: client/src/containers/Explore.js [hunk 2]
```diff
@@ -42,7 +41,7 @@ width: 0%;
   float: right;
   width: 40%;
   right: 0;
-  top: 0;
+  top: 60px;
```
Comment (Etesam913) [inline client/src/containers/Explore.js:44]: Moves the map down so it is not covered by nav

File: client/src/components/ExploreFilters/Filters.js [hunk 3]
```diff
@@ -1,101 +1,104 @@
...
-	display: flex;
-	flex-direction: row;
-	@media only screen and (max-width: 769px) {
-		display: flex;
-		flex-wrap: wrap;
-	}
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  flex-wrap: wrap;
```
Comment (Etesam913) [inline client/src/components/ExploreFilters/Filters.js:32]: Wrap's the filters down so we don't have to deal with `overflow-x` anymore

**Review by zhuci — APPROVED** [review]
LGTM!

**Pull request merged**